python/orca/src/bigdl/orca/learn/pytorch/torch_runner.py

```diff
@@ -355,18 +356,25 @@ def predict(self, partition, batch_size=32, profile=False):
                 params[arg] = config[arg]
 
         def predict_fn(shard):
-            if isinstance(shard["x"], tuple) or isinstance(shard["x"], list):
-                tensors = [torch.from_numpy(arr) for arr in shard["x"]]
+            if isinstance(partition, ray.data.Dataset):
```

Comment:
We might not introduce dependency of Ray in `torch-runner`, since Orca pyspark estimator with pyspark backend also inherits `torch_runner`